webapp/web/main.go

```diff
@@ -33,5 +34,12 @@ func main() {
 		panic(err)
 	}
 	defer shared.Clients.Close()
-	appengine.Main()
+
+	// This behaves differently in prod and locally:
+	// * Prod: the provided app.yaml is not used; it simply starts the
+	//   DefaultServerMux on $PORT (defaults to 8080).
+	// * Local: in addition to the prod behaviour, it also starts some
+	//   static handlers according to app.yaml, which effectively replaces
+	//   dev_appserver.py.
+	nicehttp.Serve("webapp/web/app.staging.yaml", nil)
```

Comment:
can we just not run it in prod (nested it under `if isDevAppserver()`)?

Reply:
We could. That was in fact my first implementation, but later I realized that `nicehttp.Serve` does basically exactly the same thing line by line except that the condition is flipped (it has its `IsProdAppEngine`). Do you prefer the explicitness?

Reply:
Right I did notice the prod check in `nicehttp.Serve`, but why do they[start a localhost:8080](https://github.com/samthor/nicehttp/blob/0bb46c93be00919c226a1bf06908e455f31f62d2/gae.go#L161) in prod?

Reply:
That's in line with https://cloud.google.com/appengine/docs/standard/go/go-differences#writing_a_main_package

Reply:
I see